main.go

```diff
@@ -49,7 +50,7 @@ func init() {
 }
 
 func handler(w http.ResponseWriter, r *http.Request) {
-	fmt.Fprintf(w, "Hello World!")
+	fmt.Fprintf(w, "Fuck You!")
```

Comment:
lol. personally i have no problem with this, but given that this is an open source project, we might want be more careful about the languages 😉

Reply:
It was just a matter of time before committing one :see_no_evil:

Reply:
https://github.com/integr8ly/workload-web-app/pull/19